### PR TITLE
Resize PolyMC logo

### PR DESCRIPTION
**Pull request by Zetvue**
I felt that it was too big for the README, so I resized it to 500px.

### CODE REVIEW

**Comment by Zetvue** [comment]
I also formatted it for no reason lol

**Review by ryanccn — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,8 +1,7 @@
-<p align="center">
-  <img src="./program_info/polymc-header-black.svg#gh-light-mode-only" alt="PolyMC logo"/>
-  <img src="./program_info/polymc-header.svg#gh-dark-mode-only" alt="PolyMC logo"/>
+ <p align="center">
+<img src="./program_info/polymc-header-black.svg#gh-light-mode-only" alt="PolyMC logo"/ width="50%">
+<img src="./program_info/polymc-header.svg#gh-dark-mode-only" alt="PolyMC logo"/width="50%">
```
Comment (topia) [inline README.md]: I don't have any preference about this change, but the original lines use self-closing tags, so please keep the slash position to last.
```suggestion
<img src="./program_info/polymc-header-black.svg#gh-light-mode-only" alt="PolyMC logo" width="50%"/>
<img src="./program_info/polymc-header.svg#gh-dark-mode-only" alt="PolyMC logo" width="50%"/>
```

**Comment by Un1q32** [comment]
@Scrumplex why does this need a milestone this is a README change.

**Comment by txtsd** [comment]
I don't actually like how small it looks with this.

**Review by ZekeZDev — APPROVED** [review]

**Review by Scrumplex — APPROVED** [review]

**Review by flowln — APPROVED** [review]
mhm it's fine either way i guess

**Pull request merged**